src/lib/setup.js

```diff
@@ -173,6 +173,17 @@ export default function(app) {
     )
   )
 
+  // Add CSRF to the cookie and remove it from the page. This will allows the
+  // caching on the html.
+  app.use((req, res, next) => {
+    const csrf = res.locals.sd.CSRF_TOKEN
+    res.cookie("CSRF_TOKEN", res.locals.sd.CSRF_TOKEN)
+    // Clear the embedded CSRF_TOKEN, an alternative method would be to update
+    // @artsy/passport to make the CSRF_TOKEN optional.
+    res.locals.sd.CSRF_TOKEN = ""
```

Comment:
So, the reason we're not removing this property from `sd` completely is that a dependency expects it (and continues to reference it, presumably?)

Reply:
I couldn't find a way to remove a property from `sd` once set and `CSRF_TOKEN` is always set in `@artsy/passport`. Maybe enhancing `sd` to support removing a property would be better. I will investigate further.

Reply:
Not a big deal. Just wanted to understand.

Reply:
Overall it's a bit odd (maybe?) that `@artsy/passport` (mounting it's middleware), is what gives Force global CSRF protection (presumably only important for those authentication server-side routes).

Although, since passport is what mounts the routes which should be protected, maybe it's ok.

Reply:
Might be able to do a `delete res.locals.sd.CSRF_TOKEN` if needed

Reply:
Funny enough, I had this exact line and it wasn't working locally. Trying it again, it seems to work fine. I was having issues with `yarn start` yesterday so maybe that was related...🤷 

What I find more odd is that in `@artsy/passport` we have `cookies: true` which requires that the `cookie-parser` middleware which is loaded in `force`